aiida/cmdline/commands/cmd_node.py

```diff
@@ -418,29 +418,37 @@ def verdi_graph():
     default='dot'
 )
 @click.option('-f', '--output-format', help="The output format used for rendering ('pdf', 'png', etc.).", default='pdf')
+@click.option(
+    '--target-cls', help='Only show nodes of target node class specified by class name', type=click.STRING, default=''
+)
 @click.option('-s', '--show', is_flag=True, help='Open the rendered result with the default application.')
 @decorators.with_dbenv()
 def graph_generate(
     root_node, link_types, identifier, ancestor_depth, descendant_depth, process_out, process_in, engine, verbose,
-    output_format, show
+    output_format, target_cls, show
 ):
     """
     Generate a graph from a ROOT_NODE (specified by pk or uuid).
     """
     # pylint: disable=too-many-arguments
+    from aiida import orm
     from aiida.tools.visualization import Graph
     print_func = echo.echo_info if verbose else None
     link_types = {'all': (), 'logic': ('input_work', 'return'), 'data': ('input_calc', 'create')}[link_types]
 
     echo.echo_info('Initiating graphviz engine: {}'.format(engine))
     graph = Graph(engine=engine, node_id_type=identifier)
     echo.echo_info('Recursing ancestors, max depth={}'.format(ancestor_depth))
+
+    origin_style_override = {'color': 'red', 'penwidth': 6}
     graph.recurse_ancestors(
         root_node,
         depth=ancestor_depth,
         link_types=link_types,
         annotate_links='both',
+        origin_style=origin_style_override,
```

Comment:
Yes, I put it here since I assume its not an option for user in cmdline interface, and I am trying to keep the internal code as much as possible, so that not to affect the default functionality. I have same consideration previously but I personally prefer the present scheme.

Reply:
When adding new features, this is indeed a good consideration, but take into account that the `origin_style` highlight is actually a current issue-bug in the code, as you noted in your OP. So I would say it makes sense that the fix for this is internal to the code, and you are making this the default behaviour, so why not make it the default `origin_style`?